docs/requirements.txt

```diff
@@ -1,3 +1,7 @@
-# Install the local crate-docs-theme package by specifying parent directory,
-# because `pip` is run from there.
---editable=.
+# Install the `crate-docs-theme` package from the local working tree
+# in editable mode. `pip` is invoked from the root directory.
+#
+# When using `uv`, don't use `--editable=.`.
+# https://github.com/astral-sh/uv/issues/10941
+
+-e .
```

Comment:
This has been fixed/improved swiftly by @charliermarsh, thank you!

- https://github.com/astral-sh/uv/issues/10941
- https://github.com/astral-sh/uv/pull/10954

Let's wait for a new release of `uv`, then revert this change again. It does not make sense to merge it this way with a misleading indication that something would not be right. Given the [current high release cadence](https://github.com/astral-sh/uv/releases), it will probably not be far away.

Reply:
Using `--editable=.` works well now, thank you Charlie!